gen/calc.py

```diff
@@ -428,11 +428,11 @@ def calculate_adminrouter_auth_enabled(oauth_enabled):
 
 
 def calculate_mesos_isolation(enable_gpu_isolation):
-    isolators = ('cgroups/cpu,cgroups/mem,cgroups/blkio,disk/du,network/cni,filesystem/linux,'
-                 'docker/runtime,docker/volume,volume/sandbox_path,volume/secret,posix/rlimits,'
-                 'namespaces/pid,linux/capabilities,com_mesosphere_MetricsIsolatorModule')
+    isolators = ('cgroups/all,disk/du,network/cni,filesystem/linux,docker/runtime,docker/volume,'
+                 'volume/sandbox_path,volume/secret,posix/rlimits,namespaces/pid,linux/capabilities,'
+                 'com_mesosphere_MetricsIsolatorModule')
     if enable_gpu_isolation == 'true':
-        isolators += ',cgroups/devices,gpu/nvidia'
```

Comment:
So, we will now enable "cgroups/devices" irrespective of "enable_gpu_isolation" which is a change in semantics. Is there any observable change in behavior that might surprise (or make it backwards incompatible for) users not enabling gpu isolation? cc @bmahler

Reply:
We have made corresponding change in the GPU isolator https://issues.apache.org/jira/browse/MESOS-9027

Reply:
So previously if GPU isolation is not enabled, the container will always have full access to all devices since `cgroups/devices` will not be enabled in this case. But now the container will only have access to the [Default whitelisted devices](https://github.com/apache/mesos/blob/1.6.0/src/slave/containerizer/mesos/isolators/cgroups/subsystems/devices.cpp#L54:L69) instead of all devices, will it be a surprising change to users?